data/human/culture conversations.txt

```diff
@@ -1627,3 +1627,270 @@ mission "A Sunracer Hovercraft Racing"
+			`	Then he playfully reveals the left card: the ace.`
+				to display
+					or
+						and
+							"monte: card spot" == 0
+							"monte: card spot" != "monte: guess"
+						and
+							"monte: guess" != 0
+							"monte: card spot" == "monte: guess"
+			`	With a single, clean motion, he shows that the middle card was the ace.`
+				to display
+					or
+						and
+							"monte: card spot" == 1
+							"monte: card spot" != "monte: guess"
+						and
+							"monte: guess" == 0
+							"monte: card spot" == "monte: guess"
+			`	He then sweeps his hand to reveal that the right card was the ace.`
+				to display
+					"monte: card spot" == 2
+					"monte: card spot" != "monte: guess"
+			branch scammed
+				"monte: card spot" == "monte: guess"
+			`	The dealer shrugs. "Better luck next time."`
+				goto again
+			label scammed
+			action
+				set "monte: scammed"
+			`	The dealer shrugs. "Better luck next time."`
+				goto again
+			label correct
+			`	And flips over an ace! The dealer seems bewildered, but begrudgingly hands over 2,000 credits, doubling your money.`
+			action
+				payment 2000
+			label again
+			branch cutoff
+				"monte: games played" < 3
+			`	He prepares to lay out a new set of cards, only to suddenly shoo you away after appearing to be spooked by some unseen authorities. The dealer quickly packs up and disappears into the spaceport.`
+				goto "true ending"
+			label cutoff
+			`	Then he gives you a look. "Fancy another game?"`
+			action
+				"monte: games played" ++
+			choice
+				`	"Of course!"`
+					to display
+						credits > 1000
+					goto "new game start"
+				`	"You cheated!"`
+					to display
+						has "monte: scammed"
+				`	"Not today."`
+					goto "true ending"
+			branch "pirate world"
+				has "flagship planet attribute: pirate"
+			`	The dealer feigns offense, shoulders lifting. "Cheated? Me? You've got quite the imagination, friend." He slides forward a fresh credit chip with a crooked grin. "Prove it. Same rules, double or nothing."`
+			choice
+				`	"Fine. But I'm watching you."`
+					to display
+							credits > 1000
+					goto "new game start"
+				`	"Sorry, but I don't have enough money."`
+					goto "true ending"
+					to display
+						credits < 1000
+				`	"And get scammed again? I don't think so."`
+					goto "true ending"
+			label "pirate world"
+			`	The dealer throws back his head and laughs. "Cheated? Sure did, what're you gonna do about it, eh?"`
+			`	From the crowd, several burly figures step forward, weapons on hips, while the dealer's grin never fades.`
+			`	"Matter of fact, you've ruined the game. That's a mark's fee, 10,000 credits. Hand it over." He jerks a thumb at the hulking men circling close. "They'll collect if you're shy."`
+			choice
+				`	"You got it, here's 10,000 credits."`
+					to display
+						credits > 10000
+				`	(Book it for my ship.)`
+					goto "book it"
```

Comment:
Where's the option to reach for my gun?

It would be interesting to have an option to intimidate/fight the pirates that has a decent chance of success with sufficient crew as backup and is a death sentence if you're alone.